extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -223,6 +226,24 @@ class AmpYoutube extends AMP.BaseElement {
       params['playsinline'] = '1';
     }
 
+    if ('loop' in params) {
+      // Loop is managed by the amp-youtube extension, don't pass it as data.
+      delete params['loop'];
+      this.user().error(
```

Comment:
Maybe `warn`. Error may flood a publishers Search Console if they are using this param (even if not working properly)

Reply:
The deprecation guide also says we have to issue a new version of the extension, not sure if that's necessary here https://github.com/ampproject/amphtml/blob/master/spec/amp-versioning-policy.md#breaking-changes

Reply:
Btw, another way to do this without breaking changes is to keep using `data-param-loop`, intercept it and not pass it to the Youtube iframe

Reply:
Great idea. let's do that! allow both `data-param-loop` and `loop` (have `loop` override `data-param-loop` in case both are present).